source/jsonnet/england-wales/communal-establishment/blocks/establishment-details/nature_of_establishment.jsonnet

```diff
@@ -0,0 +1,120 @@
+{
+  type: 'Question',
+  id: 'nature-of-establishment',
+  question: {
+    id: 'nature-of-establishment-question',
+    title: 'What is the nature of this establishment?',
+    type: 'General',
+    answers: [{
+      id: 'nature-of-establishment-answer',
+      mandatory: true,
+      options: [
+        {
+          label: 'Medical or care',
+          value: 'Medical or care',
+          description: 'For example, care homes, children’s homes or hospitals',
+        },
+        {
+          label: 'Education',
+          value: 'Education',
+          description: 'Including halls of residence',
+        },
+        {
+          label: 'Armed forces',
+          value: 'Armed forces',
+          description: 'Defence establishments, including ships',
+        },
+        {
+          label: 'Detention',
+          value: 'Detention',
+          description: 'For example, prisons or approved premises',
+        },
+        {
+          label: 'Travel or temporary accomodation',
+          value: 'Travel or temporary accomodation',
+          description: 'Includes shelters for homeless people',
+        },
+        {
+          label: 'Religious establishment',
+          value: 'Religious establishment',
+        },
+        {
+          label: 'Staff or worker accomodation only',
+          value: 'Staff or worker accomodation only',
+        },
+        {
+          label: 'Other establishment',
+          value: 'Other establishment',
+        },
+      ],
+      type: 'Radio',
+    }],
+  },
+  routing_rules: [
+    {
+      goto: {
+        block: 'medical-establishment',
+        when: [
+          {
+            id: 'nature-of-establishment-answer',
+            condition: 'equals',
+            value: 'Medical or care',
+          },
+        ],
+      },
+    },
+    {
+      goto: {
+        block: 'education-establishment',
+        when: [
+          {
+            id: 'nature-of-establishment-answer',
+            condition: 'equals',
+            value: 'Education',
+          },
+        ],
+      },
+    },
+    {
+      goto: {
+        block: 'live-in-establishment',
+        when: [
+          {
+            id: 'nature-of-establishment-answer',
+            condition: 'equals any',
+            values: ['Armed forces', 'Staff or worker accomodation only'],
+          },
+        ],
+      },
+    },
```

Comment:
Overflow doesn't state where you should be routed when they answer `Religious establishment` or  `Other establishment`. Currently, it goes to `live-in-establishment`. If those other two answers are meant to go to `live-in-establishment` then this `goto` can be removed as that is the default.

Reply:
Confirmed routing with @donnap573 - I simplified the routing rule as Religious establishment and Other establishment both route to live-in-establishment